cmd/eval.go

```diff
@@ -108,9 +121,10 @@ Output Formats
 
 Set the output format with the --format flag.
 
-	--format=json 		: output raw query results as JSON
-	--format=values 	: output line separated JSON arrays containing expression values
-	--format=bindings 	: output line separated JSON objects containing variable bindings
+	--format=json      : output raw query results as JSON
+	--format=values    : output line separated JSON arrays containing expression values
+	--format=bindings  : output line separated JSON objects containing variable bindings
+	--format=pretty    : output expression values, variable bindings, metrics, profiler results in a tabular format
```

Comment:
Nit: `"output query results in a human-readable format"`

Reply:
Done